### DISCUSSION TITLE
what's the raid performance like?

### DISCUSSION THREAD

**Discussion by ogg1e** [mebs/CM4-NAS#31] [Q&A]
Is the CM4 capable of doing any sort of raid setup, or should we just stick to standard drive mounts?

**Comment by mebs** [comment]
Raid capabilities are not dictated by the CM4 itself but by the OS or the SATA card that you use.
You could pay for a hardware raid card, but software raid works just fine. Look for `mdadm` tutorials to setup raid on linux.

**Reply by ogg1e** [reply]
sorry. I should have given you more details. I'm using the same sata card you have, and I've installed OMV, which I believe uses mdadm.

**Reply by mebs** [reply]
Oh then it's fairly straightforward, just look for raid tutorials with OMV, there are plenty. You can just use the UI.
